### PR TITLE
Added the R3 artifacts zip format

### PR DESCRIPTION
**Pull request by ounsworth**
Not to be merged until:
[x] We get the final OIDs for the NIST Draft round.
[x] We manually crawl through the existing R2 zips and extract the relevant TAs and convert them to pem, and create _r3 zips.

### CODE REVIEW

File: readme.md
Comment (johngray-dev) [inline readme.md:1]: | SPHINCS+-SHA2-128f-simple | 1.3.9999.6.4.13* | SPHINCS+-SHA2-128f-simple |
| SPHINCS+-SHA2-128s-simple | 1.3.9999.6.4.16* | SPHINCS+-SHA2-128s-simple |
| SPHINCS+-SHA2-192f-simple | 1.3.9999.6.5.10* | SPHINCS+-SHA2-192f-simple |
| SPHINCS+-SHA2-192s-simple | 1.3.9999.6.5.12* | SPHINCS+-SHA2-192s-simple |
| SPHINCS+-SHA2-256f-simple | 1.3.9999.6.6.10* | SPHINCS+-SHA2-256f-simple |
| SPHINCS+-SHA2-256s-simple | 1.3.9999.6.6.12* | SPHINCS+-SHA2-256s-simple |
| sphincsshake128fsimple | 1.3.9999.6.7.4 | sphincsshake128fsimple |
| sphincsshake192fsimple | 1.3.9999.6.8.3 | sphincsshake192fsimple |
| sphincsshake256fsimple | 1.3.9999.6.9.3 | sphincsshake256fsimple |

**Comment by dghgit** [comment]
May I suggest we also add:

- catalyst_1.2.840.10045.4.3.2_with_1.3.6.1.4.1.2.267.7.4.4_ta.pem
- catalyst_1.2.840.10045.4.3.4_with_1.3.6.1.4.1.2.267.7.8.7_ta.pem
- catalyst_1.2.840.113549.1.1.11_with_1.3.6.1.4.1.2.267.7.4.4_ta.pem
- chameleon_1.2.840.10045.4.3.2_with_1.3.6.1.4.1.2.267.7.4.4_ta.pem
- chameleon_1.2.840.10045.4.3.4_with_1.3.6.1.4.1.2.267.7.8.7_ta.pem
- chameleon_1.2.840.113549.1.1.11_with_1.3.6.1.4.1.2.267.7.4.4_ta.pem
- composite_1.2.840.10045.4.3.2_with_1.3.6.1.4.1.2.267.7.4.4_ta.pem
- composite_1.2.840.10045.4.3.4_with_1.3.6.1.4.1.2.267.7.8.7_ta.pem
- composite_1.2.840.113549.1.1.11_with_1.3.6.1.4.1.2.267.7.4.4_ta.pem

I was assuming we'd have a bit more time to do this, but it appears from Julien's email about NIST SP 1800-38 that time is not on our side, and when OQS updates we'll have even less...

The above are for:

    RSA_3072(SHA256)+Dilithium2 (security level 128 bits)
    ECDSA_P256(SHA256)+Dilithium2 (security level 128 bits)
    ECDSA_P521(SHA512)+Dilithium5 (security level 256 bits)

as given in the latest document, for each of the three hybrid types.

**Note to interested parties:** the OIDs appearing in the names for Dilithium in the names above are likely to change when OQS is updated.

**Comment by ounsworth** [comment]
Okay, I manually went through all of the `artifact.zip`'s and copied them to `artifact_certs_r3.zip`, extracting only the single TA cert for each alg to represent the simplified artifacts that we want for the r3 format.

As such though, I've touched a billion files, so the code review tab is a mess.

The files you want to review are:

* The r3 artifacts format: https://github.com/IETF-Hackathon/pqc-certificates/pull/69/files#diff-de4fd7c794655df8fa8e75f3f587a25ff30054389c00ea25d5f6c805d0888f99
* Download the raw html file for the new r3 results table and look at it in a browser: https://github.com/IETF-Hackathon/pqc-certificates/blob/1812b2d230159a1a42d5839ce7424834fb92d916/docs/pqc_hackathon_results_certs_r3.html

File: readme.md [hunk 1]
```diff
@@ -77,13 +81,53 @@ Where:
...
+Starting with artifacts for the NIST Draft standards released 2023-08-24, we will use a much simpler artifact format:
+
+* Only produce a self-signed certificate (TAs). Let's not bother with CA / EE / CRL / OCSP; those are begging for compatibility issues that have nothing to do with the PQ algs.
+* We will restrict the R3 artifacts to only the algorithms with NIST draft standards.
+* Use PEM formats.
+* Switch to a flat folder structure with filenames <oid>_ta.pem
+* For Kyber, use the the Dilithium TA of the equivalent security level to sign a <kyber_oid>_ee.pem
+
+Within `providers/<provider_name>/`
+- artifacts_certs_r3.zip
```
Comment (johngray-dev) [inline readme.md:97]: This doesn't match the OID mapping table here:  https://github.com/IETF-Hackathon/pqc-certificates/blob/master/docs/oid_mapping.md

**Pull request merged**